apparmor.d/profiles-a-f/evince

```diff
@@ -24,12 +24,15 @@ profile evince @{exec_path} {
   /{usr/,}bin/{,ba,da}sh          rix,
   /{usr/,}bin/gio-launch-desktop  rPx,
 
+  /usr/share/djvu/{,**} r,
   /usr/share/evince/{,**} r,
+  /usr/share/ghostscript/{,**} r,
   /usr/share/poppler/{,**} r,
   /usr/share/thumbnailers/{,*} r,
   /usr/share/themes/{,**} r,
 
   owner @{user_share_dirs}/ r,
+  owner @{user_share_dirs}/gvfs-metadata/{,*} r,
```

Comment:
Could this be denied? This is a major information leak and evince is purposed for deliberately risky files.

Reply:
I think it can.